src/api/Dockerfile

```diff
@@ -31,4 +31,4 @@ RUN chmod +x /app/entrypoint.sh
 # USER appuser
 USER 1001
 # ENTRYPOINT ["/app/entrypoint.sh"]
-ENTRYPOINT ["dotnet", "HSB.API.dll"]
+ENTRYPOINT ["dotnet","HSB.API.dll"]
```

Comment:
Rebase your branch, this should have already been merged by me.

Reply:
I must have chosen the wrong change during the rebase.  should be fine now.